papers/v2024/paper.md

```diff
@@ -0,0 +1,311 @@
+---
+title: 'VOTCA: multiscale frameworks for quantum and classical simulations in soft matter'
+tags:
+  - coarse-graining
+  - DFT
+  - Quantum calculations
+authors:
+  - name: Björn Baumeier^[corresponding author] # note this makes a footnote saying 'co-first author'
+    orcid: 0000-0002-6077-0467
+    affiliation: "1,2"
+  - name: Jens Wehner
+    orcid: 0000-0003-1203-5000
+    affiliation: "1,2,3"
+  - name: Nicolas Renaud
+    orcid: 0000-0001-9589-2694
+    affiliation: 3
+  - name: Felipe Zapata Ruiz
+    orcid: 0000-0001-8286-677X
+    affiliation: 3
+  - name: Rene Halver
+    orcid: 0000-0002-4895-3762
+    affiliation: 4
+  - name: Pranav Madhikar
+    orcid: 0000-0003-3658-3685
+    affiliation: "1,2"
+  - name: Ruben Gerritsen
+    orcid: 0000-0003-0738-8952
+    affiliation: "1,2"
+  - name: Gianluca Tirimbo
+    orcid: 0000-0002-6641-0761
+    affiliation: "1,2"
+  - name: Javier Sijen
+    affiliation: "1,2"
+  - name: David Rosenberger
+    orcid: 0000-0001-6620-6499
+    affiliation: 5
+  - name: Joshua S. Brown
+    orcid: 0000-0003-1227-6429
+    affiliation: 6
+  - name: Vivek Sundaram
+    orcid: 0000-0002-9986-8461
+    affiliation: "1,2"
+  - name: Jakub Krajniak
+    orcid: 0000-0001-9372-6975
+    affiliation: 7
+  - name: Marvin Bernhardt
+    orcid: 0000-0001-6520-5208
+    affiliation: 8
+  - name: Christoph Junghans^[corresponding author] # note this makes a footnote saying 'co-first author'
+    orcid: 0000-0003-0925-1458
+    affiliation: 9
+affiliations:
+  - name: Department of Mathematics and Computer Science, Eindhoven University of Technology, the Netherlands
+    index: 1
+  - name: Institute for Complex Molecular Systems, Eindhoven University of Technology, the Netherlands
+    index: 2
+  - name: Netherlands eScience Center, the Netherlands
+    index: 3
+  - name: Forschungszentrum Jülich, Jülich, Germany
+    index: 4
+  - name: Freie Universität Berlin, Berlin, Germany 
+    index: 5
+  - name: Oak Ridge National Laboratory, Oak Ridge, TN, USA 
+    index: 6
+  - name: Katholieke Universiteit Leuven, Leuven, Belgium 
+    index: 7
```

Comment:
I'm not sure how this renders without affiliation, so let's just change to:
```suggestion
  - name: Independent researcher, Poznań, Poland 
    index: 7
```
as I'm no longer affiliated with KU Leuven or any academic institution.

Reply:
Ok, will do!